tasks.py

```diff
@@ -682,6 +701,62 @@ def integration_test(
         verbose=verbose,
         append=append,
         skip_docs_build=skip_docs_build,
+        report=report,
+        generate_report=generate_report,
+    )
+
+
+@task(
+    help={
+        "keepdb": "Save and re-use test database between test runs for faster re-testing.",
+        "label": "Specify a directory or module to test instead of running all Nautobot tests.",
+        "failfast": "Fail as soon as a single test fails don't run the entire test suite.",
+        "buffer": "Discard output from passing tests.",
+        "tag": "Run only tests with the specified tag. Can be used multiple times.",
+        "exclude_tag": "Do not run tests with the specified tag. Can be used multiple times.",
+        "verbose": "Enable verbose test output.",
+        "append": "Append coverage data to .coverage, otherwise it starts clean each time.",
+        "skip_docs_build": "Skip (re)build of documentation before running the test.",
+        "report": "Generate Performance Testing report in the terminal. Set GENERATE_PERFORMANCE_REPORT=True in settings.py before using this flag",
+        "generate_report": "Generate a new performance testing report to report.json. Set GENERATE_PERFORMANCE_REPORT=True in settings.py before using this flag",
+    },
+    iterable=["tag", "exclude_tag"],
+)
+def performance_test(
+    context,
+    keepdb=False,
+    label="nautobot",
+    failfast=False,
+    buffer=True,
+    tag=None,
+    exclude_tag=None,
+    verbose=False,
+    append=False,
+    skip_docs_build=False,
+    report=True,
+    generate_report=False,
```

Comment:
This seems silly. I like `invoke performance_test` but allowing to run it without the performance tests?